x-pack/filebeat/module/cisco/asa/config/input.yml

```diff
@@ -1,8 +1,8 @@
 {{ if eq .input "syslog" }}
 
```

Comment:
I left the `input` configuration as `syslog` in order to not break existing installs. Does it makes sense, or it should align with the underlying input name?